test/ProfilingPlugin.unittest.js

```diff
@@ -15,6 +18,26 @@ describe("Profiling Plugin", () => {
 		expect(plugin.outputPath).toBe("events.json");
 	});
 
+	it("should handle outpath with folder", done => {
```

Comment:
Put this test into ProfilingPlugin.test.js as it's an Integration Test not a unit test.